validate.lic

```diff
@@ -783,14 +800,14 @@ class DRYamlValidator
   private
 
   def warn(message)
-    echo(@test_symbol) if @test_symbol
-    echo("WARNING:< #{message}  >")
+    DRC.message("#{@test_symbol}") if @test_symbol
+    DRC.message("WARNING: #{message}  ")
     @warning_count += 1
   end
 
   def error(message)
-    echo(@test_symbol) if @test_symbol
-    echo("ERROR:< #{message}  >")
+    DRC.message("#{@test_symbol}") if @test_symbol
+    DRC.message("ERROR: #{message}  ")
     @error_count += 1
   end
```

Comment:
Could the CI check be failing due to the change in how error/warning messaging is being done here?

I changed this to better highlight changes, as message vs echo causes monsterbold vs not.  I also removed the < > because this breaks monsterbold in genie (never shuts off) until it hits another popBold tag.